### PR TITLE
ui buttons changes

### PR DESCRIPTION
**Pull request by amanjr09-glitch**
The relaod and upload buttons are now seperate

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -391,3 +391,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Other Details
 Refer to the [release log](https://github.com/camicroscope/caMicroscope/releases).
+
+
+### fix
```
Comment (birm) [inline HISTORY.md:396]: Please add this as a bullet point in the latest "unreleased" code.

**Review by birm — CHANGES_REQUESTED** [review]
Hi and thanks for submitting a PR! I have just a few comments, and I've changed the target branch from "master" to "develop".

**Comment by amanjr09-glitch** [comment]
can you please elaborate a little more

**Comment by MrunalKashid02** [comment]
can i PR for this because i want customize this buttons

**Pull request closed without merging**